### PR TITLE
chore(windows/resources): Address more TODO links for help

### PR DESCRIPTION
**Pull request by darcywong00**
* Resolve [???] links
* Fix "Step 5" anchors in `start/tutorial`.  (They don't work on the site http://help.keyman-staging.com/products/windows/14.0/start/tutorial)
* Fix table Markdown formatting in character-map (I didn't address the anchors in that doc)
* Address 2nd point of keymanapp/help.keyman.com#285
> Copy desktop/current-version landing page as the top page of Windows help (in the keymanapp repo)
(See https://help.keyman.com/products/desktop/12.0/)

### CODE REVIEW

File: windows/src/desktop/help/index.md [hunk 1]
```diff
@@ -2,16 +2,51 @@
...
-* [Advanced Help](advanced/)
+## Ask for Help
+[Ask other users in the SIL Keyman Keyman Community](https://community.software.sil.org/c/keyman)
 
-* [Common Questions](common/)
+## Help Topics
 
-* [Troubleshooting](troubleshooting/)
+### [Troubleshooting](troubleshooting/)
+* [Square boxes or fonts not showing correctly](/kb/42)
```
Comment (darcywong00) [inline windows/src/desktop/help/index.md:22]: I didn't know if these should be full paths for embedded help. (To production help.keyman.com?)

Comment (mcdurdin) [reply windows/src/desktop/help/index.md:22]: Yes, should be full paths to production.

File: windows/src/desktop/help/index.md [hunk 2]
```diff
@@ -2,16 +2,51 @@
...
+* [Text Services Framework Addin](advanced/text_services_framework)
+* [Use Word macros to switch keyboard and font](/kb/33)
+* [Application Programming Interface (API)](/developer/engine/desktop/current-version/api/)
+* [Keyman for Windows version history](../version-history)
+
+### [Keyboard Layouts](/keyboard/)
+* [Tamil99](/keyboard/ekwtamil99uni)
+* [Amharic](/keyboard/gff_amharic)
+* [EuroLatin](/keyboard/sil_euro_latin)
+* [More keyboards...](/keyboard/)
```
Comment (mcdurdin) [inline windows/src/desktop/help/index.md:45]: These should also be full paths

File: windows/src/desktop/help/index.md [hunk 3]
```diff
@@ -2,16 +2,51 @@
...
+
+## Other Resources
+* [Knowledge Base](/kb)
+* [Keyman Developer Help](/developer)
+* [KeymanWeb Help](/products/web)
+* [Keyman for macOS Help](/products/mac)
+* [Keyman for Linux Help](/products/linux)
```
Comment (mcdurdin) [inline windows/src/desktop/help/index.md:52]: And these

**Review by mcdurdin — CHANGES_REQUESTED** [review]

File: windows/src/desktop/help/index.md [hunk 4]
```diff
@@ -2,16 +2,51 @@
...
+* [On Screen Keyboard](basic/toolbox/osk)
+* [Font Helper](basic/toolbox/font-helper)
+
+### [Advanced Topics](advanced/)
+* [Character Map](basic/toolbox/character-map)
+* [Hotkeys](start/hotkey_set)
+* [Text Services Framework Addin](advanced/text_services_framework)
+* [Use Word macros to switch keyboard and font](https://help.keyman.com/kb/33)
+* [Application Programming Interface (API)](https://help.keyman.com/developer/engine/desktop/current-version/api/)
+* [Keyman for Windows version history](https://help.keyman.com/products/windows/version-history)
```
Comment (darcywong00) [inline windows/src/desktop/help/index.md:39]: This won't be live to production just yet

Comment (mcdurdin) [reply windows/src/desktop/help/index.md:39]: That's probably okay.

**Review by mcdurdin — APPROVED** [review]

File: windows/src/desktop/help/index.md [hunk 5]
```diff
@@ -2,16 +2,51 @@
...
+* [Configuration Tasks](basic/config/)
+* [Language Switcher](basic/language-switcher)
+* [On Screen Keyboard](basic/toolbox/osk)
+* [Font Helper](basic/toolbox/font-helper)
+
+### [Advanced Topics](advanced/)
+* [Character Map](basic/toolbox/character-map)
+* [Hotkeys](start/hotkey_set)
+* [Text Services Framework Addin](advanced/text_services_framework)
+* [Use Word macros to switch keyboard and font](https://help.keyman.com/kb/33)
```
Comment (mcdurdin) [inline windows/src/desktop/help/index.md:37]: https://help.keyman.com/knowledge-base/93 is a newer version of this KB article

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 14.0.230-beta](https://keyman.com/downloads/releases/beta/14.0.230)

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.15-alpha](https://keyman.com/downloads/releases/alpha/15.0.15)

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.16-alpha](https://keyman.com/downloads/releases/alpha/15.0.16)

**Pull request merged**